lang/python/python3/Makefile

```diff
@@ -206,14 +205,6 @@ define Build/Compile/python3-pip
 		--root=$(PKG_BUILD_DIR)/install-pip \
 		--prefix=/usr \
 		$(PKG_BUILD_DIR)/Lib/ensurepip/_bundled/pip-$(PYTHON3_PIP_VERSION)-py3-none-any.whl
-	# FIXME: [1] get rid of this asap; 'patch' doesn't like Windows endings, and this file is full of them...
```

Comment:
We dont need this anymore?

Reply:
This workaround isn't needed (the patched file no longer contains Windows line endings).

Reply:
Okay! Thanks. Merging.